apps/builder/app/builder/features/topbar/domains.tsx

```diff
@@ -74,12 +74,13 @@ const CopyToClipboard = (props: { text: string }) => {
 };
 
 const getCname = (domain: string) => {
```

Comment:
lets add some unit tests for this @istarkov ?

Reply:
See my comment below, this will not work. I don't know how to distinguish domains like `co.za` from ordinary domains like `example.com`